testnets/junotestnet/assetlist.json

```diff
@@ -84,17 +84,13 @@
         }
       ],
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/arena.png",
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/arena.svg"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/arena.png",
```

Comment:
This image you're pointing to doesn't exist. I'd recommend having this PR only add USDC to neutron. Then in the next PR, when the new token is created, we can focus on deprecating the Juno version (not deleting it), and registering the new ARENA.

Reply:
Good to know. And you're right the images are just being moved, not deleted. I would still ask you use image_sync when you can point it to neutron's